walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +273,127 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void completelyBusyOptionalAttendee() {
+    /* Test three attendees who have different schedules, 
```

Comment:
Can you also illustrate the windows similar to previous tests so it's easier to understand and also more consistent?

Reply:
Also, previous tests use "//" style comments. Maybe also fix to be consistent.

Reply:
Addressed: 45ce2a9